server/__init__.py

```diff
@@ -396,3 +412,100 @@ def add_api_endpoints(app):
         SplitTaskAPI, "/api/v1/project/<int:project_id>/task/<int:task_id>/split"
     )
     api.add_resource(LanguagesAPI, "/api/v1/settings")
+    api.add_resource(
+        OrganisationAPI,
+        "/api/v1/organisations",
+        endpoint="create_organisation",
+        methods=["POST"],
+    )
+    api.add_resource(
+        OrganisationAPI,
+        "/api/v1/organisations/<int:organisation_id>",
+        endpoint="get_organisation",
+        methods=["GET"],
+    )
+    api.add_resource(
+        OrganisationAPI,
+        "/api/v1/organisations/<int:organisation_id>",
+        methods=["PUT", "DELETE"],
+    )
+    api.add_resource(GetAllOrganisationsAPI, "/api/v1/organisations", methods=["GET"])
+    api.add_resource(TeamAPI, "/api/v1/teams", endpoint="create_team", methods=["POST"])
+    api.add_resource(
+        TeamAPI, "/api/v1/teams/<int:team_id>", methods=["GET", "PUT", "DELETE"]
+    )
+    api.add_resource(ListTeamsAPI, "/api/v1/teams", methods=["GET"])
+    api.add_resource(
+        TeamMembersAPI, "/api/v1/teams/<int:team_id>/actions/join", methods=["POST"]
+    )
+    api.add_resource(
+        TeamMembersAPI,
+        "/api/v1/teams/<int:team_id>/actions/leave",
+        endpoint="leave_team",
+        methods=["DELETE"],
+    )
+    api.add_resource(
+        DeleteMultipleTeamMembersAPI,
+        "/api/v1/teams/<int:team_id>/actions/remove-users",
+        endpoint="remove_user_from_team",
+        methods=["DELETE"],
+    )
+    api.add_resource(
+        TeamProjectsAPI,
+        "/api/v1/teams/<int:team_id>/projects/<int:project_id>",
```

Comment:
I suggest to use `/api/v1/projects/<int:project_id>/teams/<int:team_id>` ( PUT, DELETE, PATCH)